ballerina-shell/modules/shell-core/src/test/resources/testcases/treeparser.dclnname.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name": "Input invalid function name.",
+    "input": "function main(string value) { };"
```

Comment:
Not just main. Add for the other case as well.

Reply:
Added testcases in fb1b7b1aa02f28366be706cd2d53dddafb176c76